### ISSUE TITLE
Pop up for each item in the search filters 

### ISSUE THREAD

**Issue by JannTibetan** [buda-base/public-digital-library#813]
<img width="256" alt="Screenshot 2023-03-16 at 9 13 35 PM" src="https://user-images.githubusercontent.com/44705185/225787084-ca345ca2-5583-42f3-9e8b-41ba04b2e0ce.png">

Can we replace the "bdr:03JW..." with text that defines or explains these?

**Comment by eroux** [comment]
we don't have descriptions in the topic taxonomy, just names, but yes, adding some descriptions in the 3 languages could be a good first step in the work on a new taxonomy!

If there's just the identifier we could remove the popup yes... I personally use it from time to time and I suspect other power users (such as the librarians) too, but if it confuses regular users let's not display it

**Comment by JannTibetan** [comment]
Yes, I'm thinking of it as an exercise to impose some more clarity on the taxonomy (remove redundancies and minor categories, and disambiguate the Tibetan especially [more than one use of the word sungbum, etc.])
